### PR TITLE
fix: add more metrics log for PickUpTask (v0.2.4)

### PR DESCRIPTION
**Pull request by jingjunLi**
### Description

NA

### Rationale

* move `dispatchTask` repush  into `DispatchTask`
* add more metric time for backUpTask

### Example

NA

### Changes

Notable changes: 
* NA
* ...

### CODE REVIEW

**Review by joeylichang — APPROVED** [review]

**Comment by will-2012** [comment]
Please check failed action.

File: modular/manager/manage_task.go [hunk 1]
```diff
@@ -48,13 +51,19 @@ func (m *ManageModular) DispatchTask(ctx context.Context, limit rcmgr.Limit) (ta
 			log.CtxErrorw(ctx, "dispatch task context is canceled")
 			return nil, nil
 		case dispatchTask := <-m.taskCh:
+			atomic.AddInt64(&m.backupTaskNum, -1)
 			if !limit.NotLess(dispatchTask.EstimateLimit()) {
 				log.CtxErrorw(ctx, "resource exceed", "executor_limit", limit.String(), "task_limit", dispatchTask.EstimateLimit().String())
 				go func() {
 					m.taskCh <- dispatchTask
```
Comment (will-2012) [inline modular/manager/manage_task.go:58]: `atomic.AddInt64(&m.backupTaskNum, 1)`?

File: modular/manager/manage_task.go [hunk 2]
```diff
@@ -820,3 +829,15 @@ func (m *ManageModular) pickGlobalVirtualGroup(ctx context.Context, vgfID uint32
 	log.CtxDebugw(ctx, "succeed to pick gvg", "gvg", gvg)
 	return gvg, nil
 }
+
+func GetRPCRemoteAddress(ctx context.Context) string {
```
Comment (will-2012) [inline modular/manager/manage_task.go:833]: move to util is better.

**Comment by will-2012** [comment]
```go
case <-statisticsTicker.C:
			log.CtxDebug(ctx, m.Statistics())
``` 

Add more log.

File: modular/manager/manager.go [hunk 3]
```diff
@@ -699,41 +703,50 @@ func (m *ManageModular) backUpTask() {
...
 		backupTasks = append(backupTasks, targetTask)
 	}
+	endPopTime := time.Now().String()
 
+	startPickUpTime := time.Now().String()
 	targetTask, reservedTasks = m.PickUpTask(ctx, backupTasks)
 	if targetTask != nil {
+		targetTask.AppendLog("start-pop-task-from-queue: " + startPopTime)
+		targetTask.AppendLog("end-pop-task-from-queue: " + endPopTime)
+		targetTask.AppendLog("start-pickup-task-to-dispatch" + startPickUpTime)
```
Comment (will-2012) [inline modular/manager/manager.go:713]: Refine "start-pickup-task-to-dispatch" to "start-pickup-task-to-dispatch`:`"

File: modular/manager/manager.go [hunk 4]
```diff
@@ -699,41 +703,50 @@ func (m *ManageModular) backUpTask() {
...
+		targetTask.AppendLog("end-pickup-task-to-dispatch")
```
Comment (will-2012) [inline modular/manager/manager.go:714]: Ditto.

**Review by joeylichang — APPROVED** [review]

**Pull request merged**